examples/network/nodeStyles/overwriteGroups.html

```diff
@@ -27,12 +27,10 @@
       }
     </style>
 
-    <script type="text/javascript" src="../../../dist/vis-network.js"></script>
-    <link
-      href="../../../dist/vis-network.min.css"
-      rel="stylesheet"
-      type="text/css"
-    />
+    <script
```

Comment:
in all the other examples this line is not broken up.

Reply:
The new examples that I added lately are formatted using Prettier. The old ones are unchanged and sometimes contain insanely long lines.

I personally think we should eventually go and rewrite the old (almost all) examples to give them more uniform feel and also enforce consistent style through Prettier. But this is off topic for this PR.